src/widget/OktaSignIn.js

```diff
@@ -37,13 +37,8 @@ var OktaSignIn = (function () {
       }
       if (widgetOptions.stateToken) {
         Util.introspectToken(authClient, widgetOptions)
-          .then(_.bind(function (data) {
-            var response = data;
+          .then(_.bind(function (response) {
             router = bootstrapRouter.call(this, Router, authClient, widgetOptions, renderOptions, successFn, errorFn);
-            if (isNewPipeline) {
-              response = data.rawIdxState;
-              router.appState.set('idx', data);
```

Comment:
Are we no longer setting idx on the appstate ?

Reply:
We do. See the change to `responseTransformer`